userguide/sharing.rst

```diff
@@ -972,11 +972,11 @@ possible, NTLMv1 authentication can be enabled by selecting the
 :guilabel:`NTLMv1 auth` option in
 :menuselection:`Services --> SMB -->` |ui-configure|.
 
-:numref:`Table %s <avail_vfs_modules_tab>`
-provides an overview of the available VFS modules. Be sure to research
-each module **before** adding or deleting it from the
+:numref:`Table %s <avail_vfs_objects_tab>`
+provides an overview of the available VFS objects. Be sure to research
+each object **before** adding or deleting it from the
 :guilabel:`Selected` column of the :guilabel:`VFS Objects` field of
-the share. Some modules need additional configuration after they are
+the share. Some objects need additional configuration after they are
 added. Refer to `Stackable VFS modules
```

Comment:
```suggestion
added. Refer to `Stackable VFS modules
```

This link here still refers to modules....is it supposed to?

Reply:
Yeah, unfortunately that's the title of the article on https://www.samba.org/samba/docs/old/Samba3-HOWTO/VFS.html, so I didn't think it was right to change it.

Reply:
Fair enough.